lib/cocoapods-core/platform.rb

```diff
@@ -200,6 +200,19 @@ def supports_dynamic_frameworks?
       end
     end
 
+    # @return [String] The string that describes the #symbolic_name.
+    #
+    def string_name
+      self.class.string_name(symbolic_name)
+    end
+
+    # @return [String] The string that describes the #symbolic_name,
+    #         which doesn't contain spaces and is so safe to use in
+    #         paths which might not be quoted or escaped consequently.
+    def safe_string_name
```

Comment:
ok, maybe trimmed was a bad example, but `safe` doesn't semantically convey the transformation either